Home_work_3(spiski)/Task16.py

```diff
@@ -0,0 +1,17 @@
+# Задача 16: Требуется вычислить, сколько раз встречается некоторое число X в массиве A. 
+# Пользователь в первой строке вводит натуральное число N – количество элементов в массиве. 
+# В последующих  строках записаны N целых чисел Ai (можно использовать псевдогенерацию). 
+# Последняя строка содержит число X.
+
+n = int(input("Введите количество элементов в массиве: "))
+k = int(input("Какое число ищем (в диапозоне от 1 до 10): "))
+x = 0
+import random
+
+lst = [random.randint(1,10) for i in range(n)]
```

Comment:
Если итерируемая переменная не используется в формировании значений, то вместо неё, по традиции, пишут нижнее подчёркивание.
```suggestion
lst = [random.randint(1, 10) for _ in range(n)]
```